cucumber/authenticators_jwt/features/authn_jwt_check_standard_claims.feature

```diff
@@ -68,8 +71,11 @@ Feature: JWT Authenticator - Check registered claim
       role: !group conjur/authn-jwt/raw/hosts
       member: !host myapp
     """
-    And I am the super-user
-    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    And I set the following conjur variables:
+      | variable_id                                   | default_value |
```

Comment:
@jvanderhoof I like this pattern - it makes it easier to follow and understand and it simplifies the flow